docs/pbc/all/payment-service-provider/202404.0/marketplace/stripe-third-party-integration/install-and-configure-stripe-prerequisites-for-marketplace.md

```diff
@@ -90,38 +120,183 @@ Update `config/Zed/navigation.xml` as follows:
+
+            return $conditionCollection;
+        });
+
+        return $container;
+    }
+}
+
 ```
 
 ### Add project configuration
 
-1. To configure ACL, add `'merchant-app-merchant-portal-gui'` to `\Pyz\Zed\Acl\AclConfig::addMerchantPortalInstallerRules()`.
-2. To configure the message broker, add `'merchant-commands'`, `'merchant-app-events'`, and `'app-events'` to `\Pyz\Zed\MessageBroker\MessageBrokerConfig::getDefaultWorkerChannels()`.
-3. To configure state machine, copy `vendor/spryker/sales-payment/config/Zed/Oms/StateMachine/ForeignPaymentProviderStateMachine01.xml` to the project and adjust to your needs.
+1. Configure ACL, add the following configuration `'merchant-app-merchant-portal-gui'` to
+   `\Pyz\Zed\Acl\AclConfig::addMerchantPortalInstallerRules()`:
+
+```php
+<?php
+
+namespace Pyz\Zed\Acl;
+
+use ArrayObject;
+use Generated\Shared\Transfer\RuleTransfer;
+use Spryker\Shared\Acl\AclConstants;
+use Spryker\Zed\Acl\AclConfig as SprykerAclConfig;
+
+class AclConfig extends SprykerAclConfig
+{
+    /**
+     * @var string
+     */
+    protected const RULE_TYPE_DENY = 'deny';
+
+    /**
+     * @return array<array<string, mixed>>
+     */
+    public function getInstallerRules(): array
+    {
+        $installerRules = $this->addMerchantPortalInstallerRules($installerRules);
+
+        return $installerRules;
+    }
+
+
+    /**
+     * @param array<array<string, mixed>> $installerRules
+     *
+     * @return array<array<string, mixed>>
+     */
+    protected function addMerchantPortalInstallerRules(array $installerRules): array
+    {
+        $bundleNames = [
+            'merchant-app-merchant-portal-gui',
+        ];
+
+        foreach ($bundleNames as $bundleName) {
+            $installerRules[] = [
+                'bundle' => $bundleName,
+                'controller' => AclConstants::VALIDATOR_WILDCARD,
+                'action' => AclConstants::VALIDATOR_WILDCARD,
+                'type' => static::RULE_TYPE_DENY,
+                'role' => AclConstants::ROOT_ROLE,
+            ];
+        }
+
+        return $installerRules;
+    }
+}
+```
+
+2. Configure the message broker, add `'merchant-commands'`, `'merchant-app-events'`, and `'app-events'` events:
+
+```php
+<?php
+
+namespace Pyz\Zed\MessageBroker;
+
+use Spryker\Shared\MessageBrokerAws\MessageBrokerAwsConstants;
+use Spryker\Zed\MessageBroker\MessageBrokerConfig as SprykerMessageBrokerConfig;
+
+class MessageBrokerConfig extends SprykerMessageBrokerConfig
+{
+    /**
+     * @return array<string>
+     */
+    public function getDefaultWorkerChannels(): array
+    {
+        return [
+            'app-events',
+            'merchant-commands',
+            'merchant-app-events',
+        ];
+    }
+}
+
+```
+
+3. To configure state machine, copy
+   `vendor/spryker/sales-payment/config/Zed/Oms/StateMachine/ForeignPaymentProviderStateMachine01.xml` to the project
+   and adjust to your needs.
```

Comment:
Add [cross-link](https://docs.spryker.com/docs/dg/dev/acp/integrate-acp-payment-apps-with-spryker-oms-configuration.html)  to the project OMS modification guide.